CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Extend severities with origin and date. [#192](https://github.com/greenbone/ospd-openvas/pull/192)
 - Rename the Notus Metadata Handler file to just "Metadata" [#351](https://github.com/greenbone/ospd-openvas/pull/351)
 - Check if Notus is enabled before loading metadata. [#364](https://github.com/greenbone/ospd-openvas/pull/364)
+- Launch NVTs or notus driver, depending on availability. [#366](https://github.com/greenbone/ospd-openvas/pull/366)
```

Comment:
Doesn't look like this change should go in here.

Reply:
It is a draft PR, and as you can see in my comment, depends on PR #366 . Many commits are repeated in from that PR, and after it is merged and rebased the #366, this will be fixed.